kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -164,4 +168,64 @@ public CloseableIterator<ColumnarBatch> getChangesByVersion(
 
     return DeltaLogActionUtils.readCommitFiles(engine, commitFiles, readSchema);
   }
+
+  /**
+   * Returns the raw delta actions for each version between startVersion and endVersion. Only reads
```

Comment:
There's a method `getRawChanges`, but then this method `getChanges` is documented to `Returns the raw delta actions`

Can we document / name this a bit better?